cpp/tests/centrality/betweenness_centrality_test.cu

```diff
@@ -208,7 +212,7 @@ typedef struct BC_Usecase_t {
   {
     // assume relative paths are relative to RAPIDS_DATASET_ROOT_DIR
     // FIXME: Use platform independent stuff from c++14/17 on compiler update
-    const std::string &rapidsDatasetRootDir = get_rapids_dataset_root_dir();
+    const std::string &rapidsDatasetRootDir = cugraph::test::get_rapids_dataset_root_dir();
```

Comment:
I'm assuming we're moving this to RAFT someday too? (I may already have a FIXME for that somewhere)

Reply:
I assume so.